csv-validator-core/src/test/scala/uk/gov/nationalarchives/csv/validator/api/CsvValidatorSpec.scala

```diff
@@ -60,6 +60,24 @@ class CsvValidatorSpec extends Specification with TestResources {
         case Validated.Valid(_) => ok
       }
     }
+
+    val callback = new ProgressCallback {
+      var processed = -1
+      var total = -2
+      override def update(complete: Percentage): Unit = ???
```

Comment:
should this be changed to `throw NotImplemented` or something like that instead of `???`